kobolds/biology.md

```diff
@@ -75,24 +83,34 @@ tools and creams to ease the process and to engage the help of partners in
 scratching where it is hard to reach.
 
 
-## Reproduction
+## Physiology
+
+### Reproduction
+
+Kobolds' reproductive organs are between their legs, below their bellies and in
+front of the anus (which is near the underside of the base of their tails). Most
+of the time, all individuals' reproductive organs will look like a long slit in
+the skin parallel to the spine. During arousal or for urination, this slit
+expands and opens revealing, typically, either a vulva or a penis (though of
+course genetic variation accounts for less common arrangements). Fertilization
+is internal, occurring during sex between two kobolds. In fact, there are some
+reports that fertilization may not always be required to bear young, but this is
+unconfirmed.
+
+
+### Gestation & Development
 
-Kobold young are born from eggs. Eggs are internally fertilized during sex
-between two kobolds (in fact, there are some reports that fertilization may not
-always be required, but this is unconfirmed). Eggs are laid either one or two at
-a time only and a kobold capable of laying eggs may do so as few as zero or as
-many as twelve times in their life with one to three times being very common
+Kobold young are hatched from eggs. Eggs are laid either one or two at a time
+only and a kobold capable of laying eggs may do so as few as zero or as many as
+twelve or more times in their life with one to three times being very common
 numbers of offspring.
 
 The eggs are spherical, about 6 inches in diameter and the shell is a thick,
 leathery material in a grey or milky white shade. It is slightly translucent and
-in the right lighting conditions, the growing kobold inside can be seen in vague
+in the right lighting conditions the growing kobold inside can be seen in vague
 silhouette.
 
 Gestation in the egg takes 7-8 months and temperature regulation of the egg's
 environment is key at this time. Young kobolds use already-sharp hand and foot
 claws to cut their way out of their egg when ready and then spend the next few
 months learning to move and walk and are usually speaking around a year of age.
```

Comment:
Carl, again:
> You accelerated their development at birth compared to Humans, but honestly given the advantages of external gestation I assert they would probably walk within days of birth.
>
> Which is to say, I'm invoking the theory that Humans are actually born prematurely in general because our heads already barely fit through the pelvis, and that without that constraint most things don't enter the world until they're basically capable of locomotion.

Then a whole bunch of interesting shit about eggs which, for purposes of this doc (it'll show up again in the sociology one later) boil down to: Maybe the eggs externally gestate for more than a year, which would imply the laying parent would have to imbue it with _tons_ of calories ahead of time, which means producing an egg is a high-cost biological process which probably needs some environmental trigger.

Reply:
What kind of trigger? Pheromones from other kobolds? Introductions of certain things into the diet? Needs some more research.

Reply:
Side note, but this would also explain why they could develop a culture of casual, prolific sex and not also be breeding themselves into unsustainability.